app/controllers/api/v1/workflows_controller.rb

```diff
@@ -24,6 +24,11 @@ def destroy_links
     super { |workflow| reload_queue(workflow) }
   end
 
+  def retire_subject
```

Comment:
will this accept multiple subject ids? if so maybe the name should be `retire_subjects`?